pages/knowledge/colours-and-contrast/how-to-examine/README.md

```diff
@@ -0,0 +1,43 @@
+---
+navigation_title: "How to examine"
+position: 2
+changed: "2019-04-21"
+---
+
+# How to examine colour contrast
+
+**Calculating contrast ratios is complex. Fortunately there are contrast calculators which take care of the math. We only need to provide the correct colour codes – which has a few potential pitfalls.**
+
+## Colour Contrast Analyser
+
+There are a lot of tools available to calculate the contrast between two colours. For Windows and macOS users we recommend the [Colour Contrast Analyser (CCA)](/setup/helper-tools/colour-contrast-analyser/).
```

Comment:
In-page link: rewrite to:

`(if you haven't done this yet, go back and read [Colour Contrast Analyser](/setup/helper-tools/colour-contrast-analyser/)).`